CMSgov/design-system
# [WNMGDS-2797] Merge Figma/JSON tokens feature branch

## Summary

Replaces the first TypeScript-based token storage system with [the current JSON draft standard for design tokens](https://tr.designtokens.org/f) and adds functionality to sync these tokens to and from Figma. All review has been done in the individual PRs that make up this feature branch.

## How to test

You can run various things like Storybook, doc site, and examples and just check for issues. Running the tests locally won't hurt either.

## Checklist
